bundles/org.openhab.binding.metofficedatahub/README.md

```diff
@@ -0,0 +1,463 @@
+
+It retrieves two sets of data. A hourly forecast, and daily forecast. Due to the current frequency of updates of the data retrieved,
+the hourly forecast data updates once per hour. The daily every 3 hours by default. These are configurable but likely should only need to be lowered to reduce data if needed given the point below.
+
+**IMPORTANT:** The Met Office Data Hub service is free for low volume users. Higher data usages are charged, please see their website for current information. Please bear this in mind before adjust polling rates, or adding more than 1 location (site) for forecast data, as you may need a different plan depending on the data throughput over a month, or API hit rate.
+
+Possible use case's would be to pull forecast data, for if storage heaters for example need to run for the following day, for a single site. Alternatively to start air conditioning before the day heats up, if in a old brick house, etc..
+
+In order to use this binding, you will need a Met Office Data Hub account. Once created you will need to create a plan for access to the "Site Specific" subscriptions. This will give you the client id and secret required for the bridge.
+
+## Supported Things
+
+This binding consists of a bridge for the connection to the Met Office Data Hub service, for your account. You can then add
+things to get the forecast's for a specific location (site), using this bridge.
+
+The bridge counts the total number of requests from 00:00 -> 23:59 under its properities during the runtime of the system. (This reset's if OH restarts, or the binding resets).
+
+This binding supports the follow thing types:
+
+| Thing             | Thing Type | Thing Type UID  | Discovery | Description                                                                                 |
+|-------------------|------------|-----------------|-----------|---------------------------------------------------------------------------------------------|
+| Bridge            | Bridge     | bridge          | Manual    | A single connection to the Met Office DataHub API with daily poll limiting for the Site API |
+| Site Specific API | Thing      | siteSpecificApi | Manual    | Provides the hourly and daily forecast data for a give location (site)                      |
+
+## Thing Configuration
+
+### Bridge configuration parameters
+
+| Name                       | Type   | Description                                                                                                                     | Recommended Values |
+|----------------------------|--------|---------------------------------------------------------------------------------------------------------------------------------|--------------------|
+| siteSpecificRateDailyLimit | Number | For the runtime of the system, this is the limit of how many polls for updates are allowed for updates for the SiteSpecific API | 200                |
```

Comment:
Hi @lsiepel, I believe the following is my logic behind this.

Most people shouldn't need anywhere near 200 but I set the recommendation lower than the coded default of 350 to reduce the chances of them hitting the 350 free of charge boundary. 

Technically the code is correct, but for people using the binding then the recommended value should help ensure that they don't go over the free limits. I suspect others may reboot there servers much more than I do every few months for updates, so it was put in as a recommendation rather than this is the default setting.

Hopefully that makes sense?

Reply:
Hi, I have been thinking this through quite a bit - yes its confusing. To simplify I have adjusted defaults and recommended values to 250 with a comment in the code stating this gives an extra 110 call's before the free limit is reached.